Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -422,7 +422,7 @@ def main():
     params = demisto.params()
     Client.severity = params.get('severity', '').upper()
     Client.query = params.get('query', Client.query)
-    Client.tag = params.get('tag', Client.tag)
+    Client.tag = params.get('tag', Client.tag) if params.get('tag', Client.tag) else params.get('feedTags', Client.tag)
```

Comment:
`params.get('feedTags', params.get('tag', Client.tag))`
@orhovy i think you should use something like this.